cms/templates/partials/metadata-tiles.html

```diff
@@ -40,10 +40,11 @@
         <span class="title">FORMAT</span>
         <span class="text">Online</span>
       </li>
-      {% if page.product.current_price %}
+      {% firstof page.product.first_unexpired_run.current_price or page.product.current_price as price%}
```

Comment:
Does the `firstof` tag take `or` in it ? 
Will it not work like --
`{% firstof page.product.first_unexpired_run.current_price page.product.current_price as price %}`

Reply:
good catch. It was a typo.

Reply:
Sorry, I am not getting one thing here. 
`page.product` would either be a `course` or `program`. In case of a `program` would that condition not raised an `AttributError` exception as `first_unexpired_run` is not the part of a `program` ? please correct me over it.

Reply:
No It don't, I have manually tested this change. It would be nice if you could also test it.